From 0961787d2bf0d359a3ead89e9cec642818b32dea Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Tue, 5 Jul 2022 11:51:39 +0200
Subject: [PATCH] meson.build: request native wayland-scanner

This matters in cross compilation, as otherwise meson will
try to use a cross-binary, and fail.

Upstream-Status: Submitted [https://gitlab.freedesktop.org/mesa/waffle/-/merge_requests/110]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index ca6a212..3177bde 100644
--- a/meson.build
+++ b/meson.build
@@ -110,7 +110,7 @@ else
     'wayland-egl', version : '>= 9.1', required : get_option('wayland'),
   )
   dep_wayland_scanner = dependency(
-    'wayland-scanner', version : '>= 1.15', required : get_option('wayland'),
+    'wayland-scanner', version : '>= 1.15', required : get_option('wayland'), native: true,
   )
   if dep_wayland_scanner.found()
     prog_wayland_scanner = find_program(dep_wayland_scanner.get_variable(pkgconfig: 'wayland_scanner'))
